src/Types/IndexTypes.cs

```diff
@@ -3,17 +3,20 @@
 
 namespace Pinecone;
 
+public record ListIndexesResult
```

Comment:
Perhaps you meant to mark this `internal` and place it in `Rest/Types.cs`? It's not a part of `PineconeClient` or `Index<T>` API (`ListIndexes` returns `IndexDetails[]`).

Reply:
Sounds good! yes, I wasn't sure about this one